docs/ABC_of_pull_requests.md

```diff
@@ -58,29 +70,29 @@ Next is the code review itself. Behavior is one thing, code quality is another.
 
 ***To do when:***
 
-- [ ] The code touches important core parts, is not well tested and may have side effects
+- [ ] The code touches important core parts, is not well tested and may have side effects.
 - [ ] The author specifically mentioned to test some specific thing.
 - [ ] There is some behavior change that requires validation by maintainers.
 
 If functional review is required, here are the steps to follow:
 
 - [ ] **Get the code**: There are instructions at the bottom of the PR to get the code. Click on the `command line instructions` link beside the green Merge button. See screenshot below. Refuse if there are conflicts with `dev`.
 
-![Get the code](images/PRs/getTheCode.png)
+- ![Get the code](images/PRs/getTheCode.png)
```

Comment:
There seems to be a '-' character added in front of this line. Looks like a copy-paste from the git diff...